test/postman/dbp-test-collection.json

```diff
@@ -97,7 +97,8 @@
 							"raw": ""
 						},
 						"url": {
-							"raw": "{{acsUrl}}/api/discovery",
+							"raw": "https://{{acsUrl}}/api/discovery",
```

Comment:
Lets not set the protocol here but rather in the variable definition below {{protocol}}
The build should give http or https at runtime.